### ISSUE TITLE
[NOTARY & CLIENT REMOVAL] Abusive Behaviour Detected on 1559, 1558, 1557 and 1556 from ThijsStoker

### ISSUE THREAD

**Issue by herrehesse** [filecoin-project/notary-governance#918]
## Issue Description
The individual known as ThijsStoker is not an actual person and has never engaged in legitimate transactions using the DANDI dataset on any of their applications. Despite this, notaries have knowingly signed off on these applications, allowing the storage providers to utilise this data cap in violation of the Filecoin+ program rules. Furthermore, it is important to note that none of their 20PiB (200PiB QaP) data is currently retrievable, and it is highly unlikely that it will ever be accessible. Additionally, all of the distributed data is concentrated in the same region.

## Impact
This organization's actions have enabled storage providers to experience significant growth without undertaking the necessary steps to prepare, distribute, or contribute value to the Filecoin network. This focus on sales or the expansion of their own storage providers contradicts the principles and guidelines of the Filecoin+ program, granting them unfair financial advantages over entities that are committed to operating fair and decentralized storage provider businesses. 

## Details

**Clients**

ID | Link | Title | Client | Funding Wallet
-- | -- | -- | -- | --
1559 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1559 | [DataCap Application] Distributed Archives for Neurophysiology Data Integration (DANDI)[4/4] | ThijsStoker | f13rjyv6uojeffmzthxdou36gshhq62pgoyz3y4ai
1558 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1558 | [DataCap Application] Distributed Archives for Neurophysiology Data Integration (DANDI)[4/4] | ThijsStoker | f1ts655dy5y6bv4rwus5ainj26ciphbh37eufet7a
1557 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1557 | [DataCap Application] Distributed Archives for Neurophysiology Data Integration (DANDI)[4/4] | ThijsStoker | f1prodhqvenxkb5lq555ott6hlpqgy23udm26vhdi
1556 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1556 | [DataCap Application] Distributed Archives for Neurophysiology Data Integration (DANDI)[4/4] | ThijsStoker | f1terdmbfjdrpwih67jxx5bnltyocvnf5qviww3sa

**Storage Providers**

Storage Provider | Client | IP Address | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | --
f02046115 | f02025826 | 154.210.12.64 | Chuxinxunwei Network Technology Co., Ltd. | HK , Hong Kong , Hong Kong | TRUE with fraudscore 75 | 4
f01384139 | f02025826 | 14.119.112.68 |   | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 6
f01713397 | f02025826 | 112.47.13.74 |   | CN , Guangdong , Shenzhen | FALSE with fraudscore 0 | 0
f01894158 | f02025826 | 183.178.25.228 |   | HK , Central and Western District , Central | TRUE with fraudscore 70 | 1
f02025335 | f02025826 | 116.8.132.154 |   | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 1
f02044977 | f02025826 | 115.231.235.242 |   | CN , Zhejiang , Jiaxing | TRUE with fraudscore 70 | 2
f02035545 | f02025826 | 122.102.64.18 | V pool | CN , Beijing , Beijing | FALSE with fraudscore 44 | 0
f02052410 | f02025826 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 3
f02038992 | f02025826 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 5
f01285716 | f02025826 | 115.236.46.87 |   | CN , Zhejiang , Hangzhou | TRUE with fraudscore 75 | 0
f02032240 | f02025826 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 3
f02049840 | f02025826 | 14.18.188.196 |   | CN , Guangdong , Guangzhou | TRUE with fraudscore 70 | 1
f01776264 | f02025826 | 155.254.195.143 |   | HK , Central and Western District , Central | TRUE with fraudscore 75 | 2
f02058408 | f02025826 | 103.90.153.243 |   | CN , Jilin , Huashu | TRUE with fraudscore 75 | 5
f02033556 | f02025826 | 101.71.69.98 |   | CN , Zhejiang , Ningbo | FALSE with fraudscore 0 | 7

Storage Provider | Client | IP Address | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | --
f01098931 | f02024453 | 115.236.46.87 |   | CN , Zhejiang , Hangzhou | TRUE with fraudscore 75 | 1
f02035238 | f02024453 | 115.236.46.87 |   | CN , Zhejiang , Hangzhou | TRUE with fraudscore 75 | 1
f02038992 | f02024453 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 1
f01713397 | f02024453 | 112.47.13.74 |   | CN , Guangdong , Shenzhen | FALSE with fraudscore 0 | 1
f02009761 | f02024453 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 1
f02035183 | f02024453 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 2
f01384139 | f02024453 | 14.119.112.68 |   | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 1
f02035252 | f02024453 | 110.186.65.29 | Cabrina | CN , Sichuan , Mianyang | FALSE with fraudscore 0 | 0
f02049840 | f02024453 | 14.18.188.196 |   | CN , Guangdong , Guangzhou | TRUE with fraudscore 70 | 3
f01939387 | f02024453 | 183.220.36.2 | Collen | CN , Sichuan , Chengdu | FALSE with fraudscore 0 | 5
f02041447 | f02024453 | 120.202.35.36 |   | CN , Hubei , Xiangyang | FALSE with fraudscore 0 | 6
f02034729 | f02024453 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 1
f0148494 | f02024453 | 46.19.166.57 |   | HK , Central and Western District , Central | FALSE with fraudscore 0 | 5
f0166896 | f02024453 | 112.13.70.194 |   | CN , Zhejiang , Wenzhou | FALSE with fraudscore 0 | 1
f02048047 | f02024453 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 5

Storage Provider | Client | IP Address | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | --
f02019788 | f02024424 | 49.213.6.50 |   | HK , Tsuen Wan District , Kwai Chung | FALSE with fraudscore 0 | 1
f02025335 | f02024424 | 116.8.132.154 |   | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 1
f02032240 | f02024424 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 1
f01939387 | f02024424 | 183.220.36.2 | Collen | CN , Sichuan , Chengdu | FALSE with fraudscore 0 | 2
f02038992 | f02024424 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 1
f02037983 | f02024424 | 121.14.41.152 |   | CN , Beijing , Beijing | FALSE with fraudscore 0 | 1
f02035252 | f02024424 | 110.186.65.29 | Cabrina | CN , Sichuan , Mianyang | FALSE with fraudscore 0 | 2
f02035256 | f02024424 | 183.220.36.2 | collen - f01939387 | CN , Sichuan , Chengdu | FALSE with fraudscore 0 | 0
f02041447 | f02024424 | 120.202.35.36 |   | CN , Hubei , Xiangyang | FALSE with fraudscore 0 | 7
f02048047 | f02024424 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 5
f02049840 | f02024424 | 14.18.188.196 |   | CN , Guangdong , Guangzhou | TRUE with fraudscore 70 | 5
f02056257 | f02024424 | 220.232.174.194 |   | HK , Central and Western District , Central | TRUE with fraudscore 75 | 4
f01975299 | f02024424 | 39.109.125.184 | Toploong | HK , Hong Kong , Hong Kong | FALSE with fraudscore 0 | 0
f02035183 | f02024424 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 2
f02057985 | f02024424 | 120.202.35.36 |   | CN , Hubei , Xiangyang | FALSE with fraudscore 0 | 3

Storage Provider | Client | IP Address | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | --
f02035256 | f02011870 | 183.220.36.2 | collen - f01939387 | CN , Sichuan , Chengdu | FALSE with fraudscore 0 | 1
f02038992 | f02011870 | 101.69.207.18 |   | CN , Zhejiang , Hangzhou | FALSE with fraudscore 0 | 1
f02044977 | f02011870 | 115.231.235.242 |   | CN , Zhejiang , Jiaxing | TRUE with fraudscore 70 | 1
f0148494 | f02011870 | 46.19.166.57 |   | HK , Central and Western District , Central | FALSE with fraudscore 0 | 1
f01082277 | f02011870 | 113.219.198.5 |   | CN , Hunan , Changsha | FALSE with fraudscore 0 | 1
f02032240 | f02011870 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 1
f02035183 | f02011870 | 1.182.90.5 | Mr.Liu | CN , Inner Mongolia Autonomous Region , Tongliao | FALSE with fraudscore 0 | 6
f01384139 | f02011870 | 14.119.112.68 |   | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 3
f01713397 | f02011870 | 112.47.13.74 |   | CN , Guangdong , Shenzhen | FALSE with fraudscore 0 | 2
f02033608 | f02011870 | 112.47.13.75 |   | CN , Guangdong , Shenzhen | FALSE with fraudscore 0 | 2
f01098931 | f02011870 | 115.236.46.87 |   | CN , Zhejiang , Hangzhou | TRUE with fraudscore 75 | 1
f01941622 | f02011870 | 183.220.36.2 | collen - f01939387 | CN , Sichuan , Chengdu | FALSE with fraudscore 0 | 2
f01996698 | f02011870 | 170.39.227.187 |   | US , Washington , Seattle | FALSE with fraudscore 0 | 0
f02035238 | f02011870 | 115.236.46.87 |   | CN , Zhejiang , Hangzhou | TRUE with fraudscore 75 | 1
f02041447 | f02011870 | 120.202.35.36 |   | CN , Hubei , Xiangyang | FALSE with fraudscore 0 | 2

**Notaries**

1AthSmith
1Casey-PG
1destor2023
2Meibuy
1MetaWaveInfo
1OlivierMolenkamp
1Suyanj
1TakiChain
1UnionLabs2020

1Casey-PG
2destor2023
1MetaWaveInfo
1Suyanj
2TakiChain
1UnionLabs2020

1AthSmith
1Casey-PG
1destor2023
1Meibuy
1MetaWaveInfo
1OlivierMolenkamp
2TakiChain
1UnionLabs2020

1Casey-PG
2destor2023
1MetaWaveInfo
1OlivierMolenkamp
1TakiChain
1UnionLabs2020

## Proposed Solution(s)

We propose the following actions to address the situation:
- Immediately eliminate the remaining datacap granted to the client.
- Initiate disputes against notaries who have signed inappropriately.
- Immediate removal of repeated offending notaries. **Which are all of them.**
- Blacklist the storage providers implicated in the incident.

### Timeline

Discussion at July 11th 2023 WG Call
Community open audit until July 14th 2023 at 5 pm ET
Decision announced async at 5:01 pm ET

## Additional

OlivierMolenkamp has stolen someone's identity and is already removed from the multisig.

**Comment by TijsStoker** [comment]
@herrehesse I do feel **offended** by what you said!!!

@raghavrmadya @dkkapur @galen-mcandrew Please help me with it! The man brought slander and do harm to me!

Here's the policy of Filecoin and **every one** should obey!
https://medium.com/filecoin-plus/fil-code-of-conduct-9cd044e7bcaf

>Fil+ is committed to providing a friendly, safe, and welcoming environment for all, regardless of gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics.
>
>We ask that you please respect that people have differences of opinion regarding technical choices and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer. A difference in technology preferences is not a license to be rude.
>
>Any spamming, trolling, flaming, baiting, or other attention-stealing behavior is unwelcome and will not be tolerated. Harassing other users is never tolerated, whether via public or private media.
>
>Avoid using offensive or harassing nicknames, or other identifiers that might detract from a friendly, safe, and welcoming environment for all.

**Comment by herrehesse** [comment]
@TijsStoker, I appreciate your involvement in addressing the concerns regarding your account and applications. While it is understandable that you may feel offended, it is important to acknowledge the facts regarding the abusive nature of your account and the lack of authenticity. 

In order to facilitate a mature discussion about the accusations, I kindly request that you respond with honesty and truthfulness on the following facts:

- The applications you submitted were exact replicas of our own DANDI dataset and were used in a mocking manner. There was no genuine intention to contribute value to Filecoin; your primary goal seemed to be utilising datacap. 

- The storage providers you selected for storing your dataset are unreachable, and thus there has been no verification of the presence of any DANDI data.

- You only stored your data in the same region.

- Moreover, your account name, "Tijs Stoker," is a commonly used Dutch name, similar to Olivier Molenkamp, who was previously accused and removed for identity theft. Both names were created around the same time period, and you even collaborated on signatures for your applications. 

- Lastly, it appears unusual for a genuine individual with a Dutch name to respond to my messages at 4:30 PM local time. This raises suspicions about the authenticity of your identity.

To summarise, I urge you to refrain from hiding behind claims of "slander, harm, and being offended" and instead provide me with answers to the aforementioned questions.

The notaries who signed are known to be abusive on many other applications as well:
https://github.com/filecoin-project/notary-governance/issues/907
https://github.com/filecoin-project/notary-governance/issues/892
https://github.com/filecoin-project/notary-governance/issues/882

**Comment by TijsStoker** [comment]
@herrehesse As a DP, I haven't seen any evidence about violation of the sealing rules, it is just your conjecture and attacks. I reserve the right to charge you for your words that violates the rules of community.

This is the last time I come to remind you. On the current way, there's some fatal flaw in checking vpn and retrieval success rate. None of these can be your pretext for **messing up the community and gaslighting every one**. You can find the answer in closed proposal.

**Comment by cryptowhizzard** [comment]
> @herrehesse As a DP, I haven't seen any evidence about violation of the sealing rules, it is just your conjecture and attacks. I reserve the right to charge you for your words that violates the rules of community.
> 
> This is the last time I come to remind you. On the current way, there's some fatal flaw in checking vpn and retrieval success rate. None of these can be your pretext for **messing up the community and gaslighting every one**. You can find the answer in closed proposal.

Instead of typing here i suggest you fix retrieval because as it stands now the 4 LDN's do not comply with the FIL+ rules and are eligible for removing of datacap.

**Comment by Casey-PG** [comment]
Currently the standard for retrieval is unclear, and the retrieval data provided by the robot is not yet accurate. I think we should confirm the retrieval standard first.

**Comment by cryptowhizzard** [comment]
> Currently the standard for retrieval is unclear, and the retrieval data provided by the robot is not yet accurate. I think we should confirm the retrieval standard first.

Lol 😂 

There is zero retrievability. Stop making a fool of yourself blaming others for mishandling. You shouldn’t have signed.

**Comment by TijsStoker** [comment]
@cryptowhizzard Your words and behaviors are unacceptable to the community. I've noticed that you have a bad, unexplained history of signing. https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1068#issuecomment-1373465546 
You've lost my trust.

**Comment by cryptowhizzard** [comment]
> @cryptowhizzard Your words and behaviors are unacceptable to the community. I've noticed that you have a bad, unexplained history of signing. [filecoin-project/filecoin-plus-large-datasets#1068 (comment)](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1068#issuecomment-1373465546)
> You've lost my trust.

Why not comply with the rules and make your data retrievable?

**Comment by herrehesse** [comment]
@TijsStoker can you perform KYC on your identity?

**Comment by TijsStoker** [comment]
@cryptowhizzard Why did you sign for that bad issue?

**Comment by cryptowhizzard** [comment]
> @cryptowhizzard Why did you sign for that bad issue?

This issue is not me, it’s about you. When you want to have fan time with me because you like me so much i suggest you open a new issue about me then 😂

Did you enable retrieval yet? Do you need help from community members to configure your machine?

**Comment by TijsStoker** [comment]
@cryptowhizzard Until you have a convincing explanation for your abuse, any gaslighting is invalid. https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1068#issuecomment-1373465546

**Comment by herrehesse** [comment]
I kindly request the removal of this client and the remaining datacap. Despite several accusations, no evidence has been provided, and the client has been engaging in gaslighting behaviour towards @cryptowhizzard for the past few days. There has been no progress, so I urge you to take appropriate action.

I also recommend initiating a dispute or removal process for the notaries involved, as it appears that a significant portion of them are recurring abusers (see open issues).

**Comment by TijsStoker** [comment]
@herrehesse Until you have a convincing explanation for your abuse, any gaslighting is invalid. https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1068#issuecomment-1373465546

**Comment by herrehesse** [comment]
My friend, I suggest you read through the issue: https://github.com/filecoin-project/notary-governance/issues/918#issue-1791668781

**Comment by raghavrmadya** [comment]
As @TijsStoker  has failed to respond with objective evidence, the aforementioned applications must have their DC removed. If the accused believes that the OP has violated the code of conduct kindly open a dispute
